robottelo/hosts.py

```diff
@@ -2166,6 +2166,29 @@ def enroll_ad_and_configure_external_auth(self, ad_data):
             self.execute('systemctl daemon-reload && systemctl restart httpd.service').status == 0
         )
 
+    def configure_libvirt_cr(self, server_fqdn=settings.libvirt.libvirt_hostname):
+        """Configures Libvirt ComputeResource to communicate with Satellite
+
+        :param server_fqdn: Libvirt server FQDN
+        :return: None
+        """
+        # Geneate SSH key-pair for foreman user and copy public key to libvirt server
+        self.execute('sudo -u foreman ssh-keygen -q -t rsa -f ~foreman/.ssh/id_rsa -N "" <<< y')
+        self.execute(f'ssh-keyscan -t ecdsa {server_fqdn} >> ~foreman/.ssh/known_hosts')
```

Comment:
Should you be scanning for `rsa` key here and not ecdsa ?

Reply:
ideally both works as we just need to make env-* server accessible, but here I think it'll be just using it for ssh from foreman user then ecdsa is fine, and I tried using rsa for the same purpose, it adds up unncessary warnings related to it, so I'd stick with scanning ecdsa only.